src/build_lds.ts

```diff
@@ -42,10 +43,16 @@ function copyJar() {
         rimraf.sync(config.libDirPath);
     }
     fs.mkdirSync(config.libDirPath);
-
-    // Copy the LDS jar.
-    fs.copyFileSync(config.sourceLdsJarFile,
-        path.join(config.libDirPath, config.ldsJarName))
+    const ldsJarMatches = glob.sync(config.sourceLdsJarFile);
+    if (ldsJarMatches.length > 1) {
```

Comment:
Wouldn't it be simpler to just delete all matching jars before copying a the new one? That would also obviate the need for introducing a dependency on the `glob` package, I think.

Reply:
Sure, that sounds like a good idea. We can delete the directory before we run Gradle.

This is a dev dependency, so it does not affect the size of the executable; furthermore, although we could simply copy anything that is in the directory where we expect the JAR, it is not clear to me that that would be less error-prone than comparing it either to a regex or a glob. Furthermore, it seems appropriate to me to have this error-checking because a) if the build process changes, it is good to have a sanity check (kind of like an `assert` statement), and b) unexpected things can happen in the user's file system that are outside the control of this script.

Reply:
OK, sounds reasonable. We can keep the regex, but the `-SNAPSHOT` should be replaced with a wildcard or else this won't work with releases.